### ISSUE TITLE
Offline Mathjax support

### ISSUE THREAD

**Issue by Skrylar** [dvorka/mindforger#877]
Mathjax appears to be pulled from an online service? I noticed a loading bar whenever a note is opened, and a machine without internet access wasn't resolving the formulas at all.

It's not likely that a note app should need to use a CDN to load math support, since the content is being served from the local drive.

**Comment by dvorka** [comment]
@Skrylar You are right and I don't like it as well. I really tried to make MathJax offline (I use MathJax myself heavily), but I wasn't successful. Part of the problem is that I'm JavaScript looser and I wasn't able to get an offline package/distribution that I can just load (hoping that there are no security related limitations on localhost). If you have any advice/suggestion regarding how exactly I can get fully offline version which I can run locally w/o any limitations it would really help.

**Comment by Skrylar** [comment]
What I ended up switching to was this extension for pmWiki: https://www.pmwiki.org/wiki/Cookbook/LaTeXMathJax

Which involves downloading a precompiled version of MathJax (which they offer from https://github.com/mathjax/MathJax/archive/master.zip) and slapping it in a directory accessible by the local web server. As for the cookbook plugin it looks like it just inserts a script tag, pointed toward the local web server instead of the CDN. This does however add about **twenty megabytes** of files to the server.

I don't know if you are rendering to a cache and then pointing webkit at it, doing something directly with memory or running a server and hosting through that. The way HTML actually gets to webkit probably determines how to access the local script files.

**Comment by dvorka** [comment]
@Skrylar thank you for the suggestion - this definitely helps. I will try to come with something better than MindForger has today.

**Comment by dvorka** [comment]
Implemented by MindForger 2.2.0
